packages/app/package.json

```diff
@@ -39,6 +39,7 @@
     "@roadiehq/backstage-plugin-prometheus": "^1.3.11",
     "@roadiehq/backstage-plugin-security-insights": "^1.5.11",
     "@roadiehq/backstage-plugin-travis-ci": "^1.3.10",
+    "@roadiehq/backstage-home-plugin-markdown": "^1.0.0",
```

Comment:
perhaps `backstage-plugin-home-markdown`? to be consistent with the rest

Reply:
Yea, renamed the package